d2networking/d2netpacket/d2netpackettype/message_type.go

```diff
@@ -26,3 +28,30 @@ const (
 	ServerClosed                                         // Sent by the local host when it has closed the server
 	CastSkill                                            // Sent by client or server, indicates entity casting skill
 )
+
+func (n NetPacketType) String() string {
```

Comment:
Can we make this a map?

Reply:
Absolutely, I will make that change.

I believe implementing [Stringer ](https://golang.org/pkg/fmt/#Stringer) is idiomatic, hence why Go doesn't ask us for a 
comment on that method. Out of curiosity, what is the benefit of a map?

Reply:
If we have 50 packet types, I'd rather do a hash match then a ton of CPU branches. It's not a problem at the moment, but down the road it can be.

Reply:
If anyone disagrees feel free to speak up!

Reply:
Done. I was mistaken, using a map doesn't mean not implementing Stringer, it's still implemented but references the map in place of the switch.